Source/WebCore/platform/graphics/cv/VideoFrameCV.mm

```diff
@@ -130,19 +163,8 @@
     auto source = span;
     auto destination = CVPixelBufferGetSpanOfPlane(rawPixelBuffer, 0);
     size_t bytesPerRowDestination = CVPixelBufferGetBytesPerRowOfPlane(rawPixelBuffer, 0);
-    for (unsigned i = 0; i < height; ++i) {
-        size_t j = 0;
-        while (j < std::min(plane.sourceWidthBytes, bytesPerRowDestination)) {
-            // RGBA -> ARGB.
-            destination[j] = source[j + 3];
-            destination[j + 1] = source[j];
-            destination[j + 2] = source[j + 1];
-            destination[j + 3] = source[j + 2];
-            j += 4;
-        }
-        skip(source, plane.sourceWidthBytes);
-        skip(destination, bytesPerRowDestination);
-    }
+
+    copyRGBAToARGB(source, plane.sourceWidthBytes, destination, bytesPerRowDestination, width, height);
```

Comment:
Not sure that we benefit much from a separate function for this. Consider just putting the 5 lines of code here:

```suggestion
    // Convert from RGBA to ARGB.
    auto sourceVImage = makeVImageBuffer8888(spanConstCast<uint8_t>(span), width, height, plane.sourceWidthBytes);
    auto destinationVImage = makeVImageBuffer8888(CVPixelBufferGetSpanOfPlane(pixelBuffer.get(), 0), width, height, CVPixelBufferGetBytesPerRowOfPlane(pixelBuffer.get(), 0));
    constexpr uint8_t channelMap[4] = { 3, 0, 1, 2 };
    auto error = vImagePermuteChannels_ARGB8888(&sourceVImage, &destinationVImage, channelMap, kvImageNoFlags);
    // Permutation will not fail as long as the provided arguments are valid.
    ASSERT_UNUSED(error, error == kvImageNoError);
```